README.md

```diff
@@ -12,6 +12,18 @@ In many enterprises, technology stack lock-in is a common phenomenon. For instan
 
 ## Features
 
+## 🌏  Open in the Cloud 
```

Comment:
This looks interesting, thanks for the code contribution.

This should not be in the document "Features", you could use a first level heading and put it in the appropriate place, thanks!